client/extensions/zoninator/app/controller.js

```diff
@@ -26,14 +27,16 @@ export const renderTab = ( component ) => ( context ) => {
 
 	if ( zone.length ) {
 		analyticsPageTitle += ` > ${ titlecase( zone ) }`;
-	} else {
+	}
+
+	if ( ZoneCreator === typeof component ) {
```

Comment:
As discussed via DM, let's check if `context.path` (or `.pathname` or whatever) contains `/new` rather than checking for `component`'s type.